test/functionalTest/cases/0725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -101,10 +102,11 @@ Date: REGEX(.*)
 =====================================================
 1
 0
+ftest
 --TEARDOWN--
```

Comment:
This is related with the "ftest-ftest-fest.... problem"? Which names have the TWO databases when they appear?